File: Sample/src/main/java/com/opera/ads/demo/MainActivity.java
package com.opera.ads.demo;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.opera.ads.demo.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: Sample-kotlin/src/main/kotlin/com/opera/ads/demo/MainActivity.kt
package com.opera.ads.demo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.opera.ads.demo.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.getRoot())
    }
}
